observation_portal/requestgroups/viewsets.py

```diff
@@ -254,6 +254,17 @@ def observations(self, request, pk=None):
             return Response([o.as_dict(no_request=True) for o in observations if o.state != 'CANCELED'])
         return Response([o.as_dict(no_request=True) for o in observations])
 
+    @action(detail=False, methods=['post'])
```

Comment:
The detail-oriented part of me is slightly put off by the fact that the mosaic expansion endpoint is `/api/requests/mosaic/` but the dither expansion endpoint is `/api/configuration/dither/` (plural vs. not). What do you think about making the endpoints from the configuration viewset start with `/api/configurations/` https://github.com/observatorycontrolsystem/observation-portal/blob/main/observation_portal/urls.py#L46, like all the others?

Reply:
hmm yeah I made it singular to make clear it should only take a single configuration, and we at least have one other singular endpoint with configurationstatus... But I get that the mosaic one on requests also only expects a single request. I'm fine making in plural, that'll at least be less confusing if we ever want to offer a configuration list/detail view.